pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go

```diff
@@ -31,6 +31,7 @@ const (
 
 var files = []string{
 	"crds/submariner.io_submariners_crd.yaml",
+	"crds/submariner.io_servicediscoveries_crd.yaml",
```

Comment:
This is missing the following:
1. Updates to roles/rolebindings for agent.
2. Lighthouse service account.

Reply:
I will add those in the upcoming patch, ones the agent code and roles is finalized.